Cython/Compiler/UtilNodes.py

```diff
@@ -201,7 +204,7 @@ def generate_assignment_code(self, rhs, code, overloaded_assignment=False):
         if self.type.is_pyobject:
             rhs.make_owned_reference(code)
             if not self.lhs_of_first_assignment:
-                code.put_decref(self.result(), self.ctype())
+                code.put_xdecref(self.result(), self.ctype())
```

Comment:
Seems worth a comment.

Reply:
I'm not quite sure why this was needed (or if it's still needed). It seems to pass the tests without. Possibly suggesting I should have written the comment at the time.

I've reverted it for now, but it's possible it'll become clear later...